cocos/network/WebSocket.cpp

```diff
@@ -311,33 +414,35 @@ bool WebSocket::init(const Delegate& delegate,
     {
         protocolCount = 1;
     }
-    
-	_wsProtocols = new (std::nothrow) libwebsocket_protocols[protocolCount+1];
-	memset(_wsProtocols, 0, sizeof(libwebsocket_protocols)*(protocolCount+1));
+
+    _wsProtocols = new lws_protocols[protocolCount+1];
```

Comment:
Normally, we should always add `nothrow` to new statement, riq and Wenhai have cleaned up our codes, if there is no special reason, we shouldn't remove it.